conans/test/integration/build_requires/build_requires_test.py

```diff
@@ -616,7 +616,7 @@ def test_track_host_error_nothost(self, build_profile):
         c.save({"conanfile.py":
                 GenConanfile("pkg").with_build_requirement("protobuf/<host_version>")})
         c.run(f"install . {build_profile}", assert_error=True)
-        assert "'protobuf/<host_version>': didn't find a matching host dependency" in c.out
+        assert "pkg/None require 'protobuf': didn't find a matching host dependency" in c.out
```

Comment:
Isn't this output more confusing that it was before?

Reply:
Yeah, let me fix that, I think we can make it a bit clearer 👍

Reply:
Done, please re-check